### DISCUSSION TITLE
Chromecast Support

### DISCUSSION THREAD

**Discussion by TJennerjahn** [LeoKlaus/plappa#110] [Feature Requests]
Hi,
first off, thank you for making this fantastic app.

Are there any plans on adding support for chromecast? I often use my Google home mini to listen to audiobooks and while they do support bluetooth I would prefer streaming via chromecast.

Totally understand if it's not worth the effort, but I thought I'd ask. Either way, thanks again for making this :)

**Comment by LeoKlaus** [comment]
Hey Tobias,

I've looked into this but have decided not to work on it (for now). What I personally use and can highly recommend is [AirConnect](https://github.com/philippe44/AirConnect).

It's not as fast or convenient as native Chromecast support, but in my experience it's fairly reliable and absolutely sufficient for listening to music or audiobooks.

Edit: I'm using [this container](https://hub.docker.com/r/1activegeek/airconnect), in case you want to check it out.
